README-permission.md

```diff
@@ -153,10 +174,12 @@ Variable | Description | Required
 `object_type` | Type of IPA object (sets subtree and objectClass targetfilter) | no
 `no_members` | Suppress processing of membership | no
 `rename` | Rename the permission object | no
-`privilege` | Member Privilege of Permission | no
 `action` | Work on permission or member level. It can be on of `member` or `permission` and defaults to `permission`. | no
 `state` | The state to ensure. It can be one of `present`, `absent`, or `renamed` default: `present`. | no
 
+The `includedattrs` and `excludedattrs` variables are not exposed by the module as also `attrs` can be used in these cases and `includedattrs` and `excludedattrs` are set internally automatically.
```

Comment:
What about: "The `includedattrs` and `excludedattrs` variables are not exposed by the module as `attrs` can be used in these cases, providing complete control over attributes with a simpler interface. When required, `includedattrs` and `excludedattrs` are set internally."

Reply:
I have replaced this with "The `includedattrs` and `excludedattrs` variables are only usable for managed permisions and are not exposed by the module. Using `attrs` for managed permissions will result in the automatic generation of `includedattrs` and `excludedattrs` in the IPA server."